documentation/templates/documentation/partials/block.html

```diff
@@ -166,7 +166,12 @@ <h2>Tips</h2>
 {% if code_block.ext_doc %}
     <div id="external_doc">
         <h2>Additional Information</h2>
-        <p>From <a href="{{ code_block.ext_doc }}" taraget="_blank">{{ code_block.ext_doc }}</a></p>
-        <iframe src="{{ code_block.ext_doc }}" width="100%" height="1000px" frameborder="none"></iframe>
+        <!-- dont render iframe for w3schools content, see https://codedotorg.atlassian.net/browse/LP-1424 -->
+        {% if 'w3schools' in code_block.ext_doc %} 
+          <p>For more information, see <a href="{{ code_block.ext_doc }}" taraget="_blank">{{ code_block.ext_doc }}</a></p>
```

Comment:
looks like the original code had a typo in "target"

Reply:
Good catch!